.gitignore

```diff
@@ -1,5 +1,6 @@
 htmlerror
 *~
+.idea/*
```

Comment:
For future reference: you can gitignore your editor's files globally with core setting `excludesfile`.